src/tests/class_tests/openms/source/MzIdentMLFile_test.cpp

```diff
@@ -327,8 +339,8 @@ START_SECTION(([EXTRA] regression test for file loading on example files))
   std::vector<PeptideIdentification> peptide_ids;
   String input_path = OPENMS_GET_TEST_DATA_PATH("MzIdentMLFile_whole.mzid");
   MzIdentMLFile().load(input_path, protein_ids, peptide_ids);
-  input_path = OPENMS_GET_TEST_DATA_PATH("Mascot_MSMS_example.mzid");
-  MzIdentMLFile().load(input_path, protein_ids, peptide_ids);
+//  input_path = OPENMS_GET_TEST_DATA_PATH("Mascot_MSMS_example.mzid");
```

Comment:
Because it uses an old example mzid file from the Mascot guys... and it has some flaws. And that is why I greyed it out and did not delete it, because we can use it for flawed input testing